client/src/partials/reports/balance_mensuelle/balance_mensuelle.html

```diff
@@ -3,43 +3,41 @@
     <main class="extend">
       <div class="panel panel-default">
         <div class="panel-heading">
-          <b> {{ "REPORT.CONFIGURE_REPORT" | translate }} : </b> {{ "BALANCE_MENSUELLE.TITLE" | translate }}
+          <strong>{{ "REPORT.CONFIGURE_REPORT" | translate }}:</strong> {{ "BALANCE_MENSUELLE.TITLE" | translate }}
```

Comment:
This could be much better represented by heading tag (h1...6) but this idea can be addressed in the bootstrap upgrade.